docs/tutorials/devel/install_devmode/index.txt

```diff
@@ -45,6 +50,33 @@ For Windows: (:ref:`_install_win_devmode`)
 
     $ sudo pip install virtualenvwrapper
 
+   *Postgresql*
+
+     .. code-block:: console
+
+      $ sudo apt-get install postgresql-9.3-postgis-2.1 postgresql-9.3-postgis-scripts
+
+     Change postgres UNIX password
+
+     .. code-block:: console
+
+      $ sudo passwd -u postgres # change password expiry infromation
```

Comment:
Postgresql doesn't ask users to set password during initial install on Ubuntu, so its necessary to create new password for postgres. Though one can also user thier default system user, but postgres will through an error, that database doesn't exist.